test/unit/helpers/govspeak_helper_test.rb

```diff
@@ -475,4 +475,19 @@ class GovspeakHelperTest < ActionView::TestCase
     html = govspeak_with_attachments_to_html(body, attachments, "batman@wayne.technology")
     assert html.include? ">batman@wayne.technology</a>"
   end
+
+  test "converts Brexit CTA govspeak to HTML" do
+    body = "$BrexitCTA\nSome other text"
+    output = govspeak_to_html(body)
+    paragraph_text = "The UK is leaving the EU. This page tells you how to "\
```

Comment:
This is quite coupled to the callout box as it is now. If that is changed this test breaks. If we extract the html in a file, we could check if the govspeak gets replaced by the contents of that file

Reply:
This is a tricky one, because the output of `govspeak_to_html` is HTML, whereas as the content of the file would be markdown, so I think we will need to check for some HTML.  If the content of the CTA does change I don't think it would be too much effort to update the test.

Reply:
true, it was a minor thing - hopefully the line you're checking won't change 👍